components/x-eventpromo/src/Meta.jsx

```diff
@@ -0,0 +1,12 @@
+import { h } from '@financial-times/x-engine';
+
+const Meta = ({location, dates}) => {
+	return (
+		<div className="event-promo__meta">
+			<p>{location}</p>
```

Comment:
Should this be an `<address>` element or is it just a word or two?

Reply:
It's just free text, it can be anything. 
Usually it's something like, "London", "New York", "ExCeL London", etc

Reply:
It is likely to just be a place name and City at most but I don't see any harm in using <address>